PostgreSQL.md

```diff
@@ -138,7 +140,23 @@ To activate this feature, do one of the following:
 * set the `WALG_USE_REVERSE_UNPACK`environment variable
 * add the --reverse-unpack flag
 ```
-wal-g backup-fetch ~/extract/to/here LATEST --reverse-unpack
+wal-g backup-fetch /path LATEST --reverse-unpack
+```
+
+#### Redundant archives skipping
+
+With [reverse delta unpack](#reverse-delta-unpack) turned on, you also can turn on redundant archives skipping.
+Since this feature involves both backup creation and restore process, in order to fully enable it you need to do two things:
+
+1. Optional, but recommended. [Enable rating tar ball composer](#rating-composer-mode) for `backup-push`.
+
+2. Enable redundant backup archives skipping during backup-fetch. Do one of the following:
+  
+* set the `WALG_USE_REVERSE_UNPACK` and `WALG_SKIP_REDUNDANT_TARS` environment variables
+* add the `--reverse-unpack` and `--skip-redundant-tars` flags
```

Comment:
Does it work without --reverse-unpack?

Reply:
It will technically work but it won't skip any tarballs so it will be useless